src/bool.satyg

```diff
@@ -1,11 +1,71 @@
+@import: base
+@import: ord
+@import: eq
+@import: float
+
 module Bool : sig
+  val ord : (bool Ord.t) implicit
+  val eq : (bool Eq.t) implicit
+
+  val equal : bool -> bool -> bool
+
+  val (<) : bool -> bool -> bool
+  val (>) : bool -> bool -> bool
+  val (>=) : bool -> bool -> bool
+  val (<=) : bool -> bool -> bool
+  val (==) : bool -> bool -> bool
+
+  val max : bool -> bool -> bool
+  val min : bool -> bool -> bool
+  val max-value : bool
+  val min-value : bool
 
+  val to-int : bool -> int
+  val of-int-opt : int -> bool option
+  val to-float : bool -> float
+  val of-float-opt : float -> bool option
```

Comment:
I don't think to-float and of-float-opt are useful in practice.
I would not include them.

Reply:
I agree.